docs/offer/offer.yaml

```diff
@@ -310,3 +310,59 @@ paths:
                 status: 404
                 code: OFFER_NOT_FOUND
                 message: Offer with provided data was not found.
+  /offers/price-range{?authorRole}:
```

Comment:
remove {?authorRole} here, you define it in the query params, not url

Reply:
fixed